numpy/core/code_generators/generate_umath.py

```diff
@@ -1137,6 +1144,13 @@ def english_upper(s):
           TD(O),
           signature='(n?,k),(k,m?)->(n?,m?)',
           ),
+'isalpha' :
```

Comment:
Just to make sure: this only shows up in the internal `ufunc` namespace?  (actually I guess tests would fail if not)

Reply:
That's correct, yes.